components/TripPreview.js

```diff
@@ -1,28 +1,31 @@
 import styled from "styled-components";
 import Image from "next/image";
+import Link from "next/link";
 
 export default function TripPreview({ trip }) {
   return (
     <StyledArticle>
-      <StyledImageWrapper>
-        <StyledImage
-          src={trip.image}
-          height={800}
-          width={800}
-          alt={trip.location}
-        />
-      </StyledImageWrapper>
-      {trip.location}
-      <br />
-      {`${trip.startDate.substring(8)}/${trip.startDate.substring(
-        5,
-        7
-      )}/${trip.startDate.substring(2, 4)}`}{" "}
-      -{" "}
-      {`${trip.endDate.substring(8)}/${trip.endDate.substring(
-        5,
-        7
-      )}/${trip.endDate.substring(2, 4)}`}
+      <StyledLink href="/explore">
+        <StyledImageWrapper>
+          <StyledImage
+            src={trip.image}
+            height={800}
+            width={800}
+            alt={trip.location}
+          />
+        </StyledImageWrapper>
+        {trip.location}
+        <br />
+        {`${trip.startDate.substring(8)}/${trip.startDate.substring(
+          5,
+          7
+        )}/${trip.startDate.substring(2, 4)}`}{" "}
+        -{" "}
+        {`${trip.endDate.substring(8)}/${trip.endDate.substring(
+          5,
```

Comment:
you could create a small utilityfunction that takes the date string and transforms it, your code would be more readable then. But thats only optional